sites/flanco.py

```diff
@@ -0,0 +1,51 @@
+from scraper_peviitor import Scraper
+from utils import (
+    publish,
+    publish_logo,
+    show_jobs
+)
+from getCounty import get_county
+
+url = "https://www.flanco.ro/cariere/posturi-vacante"
+
+final_jobs = []
+company = "flanco"
+
+scraper = Scraper(url=url)
+
+while True:
+    scraper.getSoup()
+    content = scraper._soup
+
+    jobs = content.select('div.vacant-position')
+    for job in jobs:
+        job_info = {
+            "title": job.select_one("div.vp-title h2").get_text(),
+            "link": job.select_one("div.description div.buttons a").get("href"),
```

Comment:
cheile title si link nu sunt valide 
Asa arata un obiect valid.
pot fi stringuri sau liste.
![Screenshot 2024-02-25 at 20 11 16 (2)](https://github.com/peviitor-ro/based_scraper_py/assets/67306273/a1e6c830-4c09-4a1f-96f8-85491b14993b)

```suggestion
            "job_title": job.select_one("div.vp-title h2").get_text(),
            "job_link": job.select_one("div.description div.buttons a").get("href"),
```